plugins/woo-ai/src/hooks/useProductDataSuggestions.ts

```diff
@@ -25,15 +25,16 @@ export const useProductDataSuggestions = () => {
 		request: ProductDataSuggestionRequest
 	): Promise< ProductDataSuggestion[] > => {
 		try {
-			const response = await requestJetpackToken().then( ( { token } ) =>
-				apiFetch< ProductDataSuggestionSuccessResponse >( {
-					path: '/wooai/product-data-suggestions',
-					method: 'POST',
-					data: { ...request, token },
-				} )
+			const { suggestions } = await requestJetpackToken().then(
```

Comment:
Perhaps nitpicky but what about a more consistent approach to Promises (either using async/await or promise chains but not both)?
I like async/await because it's more readable:

```
  const { token } = await requestJetpackToken();
  
   const { suggestions }  = await apiFetch<ProductDataSuggestionSuccessResponse>({
    path: '/wooai/product-data-suggestions',
    method: 'POST',
    data: { ...request, token },
  });
```

Reply:
> Also, if you have a longer chain it actually seems more readable to be that way, although with just the two requests above I can see it feeling less readable.
> 
> ```js
> const finalResult = await doThing().then(doAnotherThing)
> 			.then(doYetAnotherThing)
> 			.then(doTheLastThing)
> 			.catch(e -> handleError);
> ```

That can be true for sure, and I do love the concept of method chaining. 😆 
Although, the counter-argument is that it makes debugging quite a lot more difficult, right?
And, while it is super practical to put a `.catch()` at the end to catch everything, you can accomplish the same with `try{} catch{}` blocks. 🤔 

And just as an FYI, from time to time I have recommendations/questions like this which aren't really meant as criticisms but rather serve as learning experiences for me, given that I don't have a _ton_ of experience, so it's helpful for me to hear different opinions.  🙇

Reply:
> And just as an FYI, from time to time I have recommendations/questions like this which aren't really meant as criticisms but rather serve as learning experiences 

For sure! They're potential learning experiences for both of us, and it's always good to question those things that I'm just "in the habit" of doing (and not always certain why). Keep'm coming!